news/2018-11-21-new-featured-artist-culprate.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Featured Artist: Culprate"
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom to the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
+
+One of **Culprate**'s most recent projects is a massive multi-work collaboration drawn from his Twitch community, of which one of our Featured Artists (namely [Dictate](https://osu.ppy.sh/beatmaps/artists/21)) was a part of. Aptly titled as the *Unity Project*, we are proud to announce that a selection from this body of work and from **Culprate**'s cult classic album *Deliverance* are now available.
```

Comment:
"of which one of our featured artists was a part of" is really heavy and uncomfortable to read. Try to reword it?